From 0a32f736ae2e7705e40bb9467edc208345915069 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 22 Jan 2019 15:50:52 +0100
Subject: [PATCH 6/6] Don't export BUILD_NUMBER in envsetup

* The value is meant to be set only before running an /official/ build
* This allows us to execute consecutive make commands without
  running kati for every invocation

Change-Id: Ifa94b839f30c7260009c8a3c91c202b50e28022e
---
 envsetup.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/envsetup.sh b/envsetup.sh
index d63a50676..910c3ce8d 100644
--- a/envsetup.sh
+++ b/envsetup.sh
@@ -139,7 +139,6 @@ function check_product()
     fi
     if (echo -n $1 | grep -q -e "^lineage_") ; then
         LINEAGE_BUILD=$(echo -n $1 | sed -e 's/^lineage_//g')
-        export BUILD_NUMBER=$( (date +%s%N ; echo $LINEAGE_BUILD; hostname) | openssl sha1 | sed -e 's/.*=//g; s/ //g' | cut -c1-10 )
     else
         LINEAGE_BUILD=
     fi
-- 
2.17.1

